radicle-cli/src/commands/assign.rs

```diff
@@ -0,0 +1,96 @@
+use std::ffi::OsString;
+use std::str::FromStr;
+
+use anyhow::anyhow;
+
+use crate::terminal as term;
+use crate::terminal::args;
+use radicle::cob;
+use radicle::cob::issue;
+use radicle::storage::WriteStorage;
+
+pub const HELP: args::Help = args::Help {
+    name: "assign",
+    description: "assign an issue",
+    version: env!("CARGO_PKG_VERSION"),
+    usage: r#"
+Usage
+
+    rad assign <issue> <peer>
+
+Options
+
+    --help      Print help
+"#,
+};
+
+#[derive(Debug)]
+pub struct Options {
+    pub id: issue::IssueId,
+    pub peer: cob::ActorId,
+}
+
+impl args::Args for Options {
+    fn from_args(args: Vec<OsString>) -> anyhow::Result<(Self, Vec<OsString>)> {
+        use lexopt::prelude::*;
+
+        let mut parser = lexopt::Parser::from_args(args);
+        let mut id: Option<issue::IssueId> = None;
+        let mut peer: Option<cob::ActorId> = None;
+
+        while let Some(arg) = parser.next()? {
+            match arg {
+                Long("help") => {
+                    return Err(args::Error::Help.into());
+                }
+                Value(ref val) => {
+                    if id.is_none() {
+                        let val = val.to_string_lossy();
+                        let Ok(val) = issue::IssueId::from_str(&val) else {
+                            return Err(anyhow!("invalid issue ID '{}'", val));
+                        };
+
+                        id = Some(val);
+                    } else if peer.is_none() {
+                        let val = val.to_string_lossy();
+                        let Ok(val) = cob::ActorId::from_str(&val) else {
+                            return Err(anyhow!("invalid peer ID '{}'", val));
+                        };
+
+                        peer = Some(val);
+                    } else {
+                        return Err(anyhow!(arg.unexpected()));
+                    }
+                }
+                _ => {
+                    return Err(anyhow!(arg.unexpected()));
+                }
+            }
+        }
+
+        Ok((
+            Options {
+                id: id.unwrap(),
+                peer: peer.unwrap(),
+            },
+            vec![],
+        ))
+    }
+}
+
+pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
+    let profile = ctx.profile()?;
+    let signer = term::signer(&profile)?;
+    let storage = &profile.storage;
+    let (_, id) = radicle::rad::cwd()?;
+    let repo = storage.repository(id)?;
+    let mut issues = issue::Issues::open(*signer.public_key(), &repo)?;
+
+    let mut issue = issues.get_mut(&options.id).map_err(|err| match err {
+        cob::store::Error::NotFound(_, _) => anyhow!("issue not found '{}'", options.id),
+        _ => err.into(),
+    })?;
+    issue.assign(vec![options.peer], &signer)?;
+
+    Ok(())
```

Comment:
What's our general policy on finishing with `term::success`? I noticed that we use it some places like `track` or `patch`, but not in `issue`.

Not a blocker for this, but I'm curious :)

Reply:
I am curious too, maybe we can codify these in the crates documentation?

Reply:
"finished" commands will usually end with `term::success`, but the issue commands haven't been polished yet, hence it's just the functionality and none of the visuals.